### DISCUSSION TITLE
What does "A=" mean in the Makefile?

### DISCUSSION THREAD

**Discussion by lizhi16** [unikraft/unikraft#452] [Q&A] [answered]
Hello, we study the Makefile and find that the parameter "A=" in the Makefile is uncommon for us. We studied the "make --help", but didn't find any explanation. 

What is the function of the "A=" in the Makefile? Thanks!

**Comment by nderjung** [comment, marked as answer]
Hi @lizhi16,

`A=` is the path to the application source (aka the source files of the application repository, e.g. `app-helloworld`) when set up correctly.  The command you ran, `make --help` invokes GNU Make's help menu instead of Unikraft.  For Unikraft's help menu, please try `make help` (without the `--`) as this is a unique target within Unikraft.

**Reply by lizhi16** [reply]
Thanks for your answer! It is very helpful for us.

**Comment by razvand** [comment]
@lizhi16, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
